assets/src/components/scheduleAdherence.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ComponentPropsWithoutRef, useContext } from "react"
```

Comment:
question (non-blocking): Apologies for not noticing this before! It seems that there is overlap between this component and what is in `propertiesPanel/header.tsx` - is there enough overlap that `header` can be updated to use this component for consistency?

Reply:
Nevermind! when trying to do this it broke in a lot of ways(spacing, flex container, alignment), I think we're better off waiting for design to have us replace the header in some form.